src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -7,7 +7,7 @@
 use super::vmm_instance::VmmInstance;
 use crate::{
     device::DeviceType, hypervisor_persist::HypervisorState, kernel_param::KernelParams, VmmState,
-    DEV_HUGEPAGES, HUGETLBFS, HUGE_SHMEM, HYPERVISOR_DRAGONBALL, SHMEM,
```

Comment:
If I understand your question correctly, they are not removed actually. They are just moved from `src/lib.rs` to `dragonball/inner.rs` other than `HUGETLBFS` The moved constants are only used by dragonball, which gave me the `never used` compile error when the hypervisor is excluded from the compilation on s390x. `HUGETLBFS` is left in `src/lib.rs` as it was because it is used by the `resource` crate. Any suggestion for leaving them as they were while not facing with the compile error would be appreciated. Thanks.

Reply:
@BbolroC thanks, I understand